README.md

```diff
@@ -31,22 +40,55 @@ Name|Value Type|Example Value| Required|Default
 ----|----------|-------------|---------|-------------
 log-level|string|"warn"|false|"info"
 github-token|string| |true|null
-jira-user|string|"user@jira.example.com"|true|null
-jira-pass|string|"theUserPassword123"|true|null
+jira-user|string|"user@jira.example.com"|false|null
+jira-pass|string| |false|null
+jira-token|string| |false|null
+jira-secret|string| |false|null
+jira-consumer-key|string| |false|null
+jira-private-key|string| |false|null
 repo-name|string|"coreos/issue-sync"|true|null
 jira-uri|string|"https://jira.example.com|true|null
 jira-project|string|"SYNC"|true|null
 since|string|"2017-07-01T13:45:00-0800"|false|"1970-01-01T00:00:00+0000"
+timeout|duration|5000000000|false|60000000000
```

Comment:
I've been trying to figure out how to, but I haven't found anything. Cobra does seem to interpret it correctly as input, but I don't know how to get it to save it in that sort of format.